package.json

```diff
@@ -33,29 +33,29 @@
     "test:xunit": "tape 'test/unit/**/*.test.js' | tap-xunit",
     "test:coverage": "nyc --reporter=lcov --reporter=text-summary tapes -- 'test/unit/**/**.test.js'",
     "test:coverage-check": "npm run test:coverage && nyc check-coverage",
-    "audit:resolve": "SHELL=sh resolve-audit --production",
-    "audit:check": "SHELL=sh check-audit --production",
+    "audit:fix": "npm audit fix",
+    "audit:check": "npx audit-ci --config ./audit-ci.jsonc",
     "dep:check": "npx ncu -e 2",
     "dep:update": "npx ncu -u",
     "release": "standard-version --releaseCommitMessageFormat 'chore(release): {{currentTag}} [skip ci]'"
   },
   "dependencies": {
-    "knex": "0.95.5",
+    "knex": "2.5.1",
     "lodash": "4.17.21",
     "mysql": "2.18.1"
   },
   "devDependencies": {
-    "faucet": "0.0.1",
+    "audit-ci": "^6.6.1",
+    "faucet": "0.0.4",
```

Comment:
Lets replace`faucet` with `tap-spec`, since `faucet` was deprecated.

See https://github.com/mojaloop/ml-api-adapter/blob/main/package.json for reference.

Reply:
Take note that this will impact the `test:unit` npm scripts.